.github/scripts/dpf_clap_validate.py

```
@@ -0,0 +1,191 @@
+        sys.exit(2)
+    clap = sys.argv[1]
+    if not os.path.exists(clap):
+        log(f"::error::plugin not found: {clap}")
+        sys.exit(2)
+
+    workdir = tempfile.mkdtemp(prefix="clapval_")
+    try:
+        vbin = resolve_binary(workdir)
+        log(f"Using clap-validator: {vbin}")
+        try:
+            proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)